backend/gamecreation/views.py

```
@@ -6,35 +6,243 @@
+        importValue=request.data.get('importValue')
+        api=ApiKeys.objects.create(user=user,apikey=request.data['apiToken'],cloudsite=request.data['site'])
+        board=PokerBoard.objects.create(game_name=request.data['gameName'],game_description=request.data['gameDescription'])   
+        pokermember.objects.create(poker=board, member=user, role=3)
+        for userinfo in request.data.get('users'):
+            if (not userinfo.get('invite')):
+                pokermember.objects.create(poker=board, member_id=userinfo.get('id'), role=userinfo.get('role'))
+            else :
+                Invite.objects.create(pokerboard=board,host=user,guest=userinfo.get('email'),accept=False,role=userinfo.get('role'))
         
+        if(importType=='tickets'):
+            for Ticket in request.data.get('tickets'):
```

Ruff (0.12.2)

Loop control variable `Ticket` not used within loop body

Rename unused `Ticket` to `_Ticket`

(B007)